contributors/guide/issue-triage.md

```diff
@@ -4,57 +4,233 @@ weight: 10
+- label other SIGs, if appropriate
+- discuss briefly the technical implications
+- assign people with expertise in the domain to review, comment, reject, etc.
+
+api-machinery has found that consistently meeting on a regular, fixed schedule is key to the success of a triaging effort. More frequent, small meetings are better than infrequent, large meetings, they've found. A few other pointers:
+- We try to balance the load, and ask people if they are okay taking on an issue before assigning it to them
+- We skip issues that are closed
+- We also skip cherrypicks, because we consider that the code change was reviewed in the original PR
+- We ensure participation from the entire SIG and support company diversity. 
+- We use this opportunity to mark ["help needed", "good first issue"](#help-wantedgood-first-issues).
+
+### Triage Guide by cluster-lifecycle
+has developed a [triaging page](/sig-cluster-lifecycle/grooming.md) detailing their process, including the [Milestones](#planning-milestones) stage. Here is a [March 2020 presentation](https://www.youtube.com/watch?v=Q07_PfkNjlw) delivered to the SIG chairs and leads group on their process.
 
-These guidelines serves as a primary document for triaging incoming issues to
-Kubernetes. SIGs and projects are encouraged to either use these guidelines, or
-use this as a starting point if necessary. For example, if your SIG has specific
-triaging needs, extend these guidelines.
-**Note:** These guidelines only applies to the Kubernetes repository. Its usage
-for other GitHub repositories related to Kubernetes is TBD.
+## Step One: Review Newly Created Open Issues
+Kubernetes issues are listed [here](https://github.com/kubernetes/kubernetes/issues). New, untriaged issues come without labels attached. SIG leads should identify at least one SIG member to serve as a first point of contact for new issues.
 
-## Using the bot
+Labels are the primary tools for triaging. [Here's a comprehensive label list](https://github.com/kubernetes/kubernetes/labels).
 
-Most people can leave comments and open issues. They don't have the ability to
-set labels, change milestones and close other people's issues. For that we use
-a bot to manage labelling and triaging. The bot has a set of
-[commands and permissions](https://go.k8s.io/bot-commands)
-and this document will cover the basic ones.
+### Conducting Searches
+GitHub allows you to filter out types of issues and pull requests, which helps you discover items in need of triaging. This table includes some predetermined searches for convenience:
 
-## Determine if it's a support request
+|  Search | What it sorts  |
+|---|---|
+| [created-asc](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-asc)  | untriaged issues by age |
+|  [needs-sig](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+label%3Aneeds-sig) | issues that need to be assigned to a SIG  |
```

Comment:
Hi: This prompt only directs to issues, but not PRs, labeled `needs-sig`. Should PRs be included in the search query as well?

Reply:
GitHub's UI really only lets you do issues or PRs when viewing them =/ Id default to viewing PRs as those are the higher priority items to triage.